jetty-util/src/main/java/org/eclipse/jetty/util/ReadLineInputStream.java

```diff
@@ -22,6 +22,9 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
+import java.util.EnumSet;
+
+import org.eclipse.jetty.util.MultiPartInputStreamParser.NonCompliance;
 
 /**
  * ReadLineInputStream
```

Comment:
Deprecate this steaming pile of code also

Reply:
done